templates/components/itilobject/service_levels.html.twig

```diff
@@ -162,7 +162,8 @@
                   </div>
                {% endif %}
 
-               {% if la_displayed %}
+               {% if (la_displayed) or (item.fields[la_field.lafieldname] != 0) %}
```

Comment:
This condition should be moved in the `la_displayed` variable definition, ortherwise, if the date is hidden too, the entire block will not be displayed due to the `{% if date_displayed or la_displayed %}` check on line 85.